src/ErrorHandler.php

```diff
@@ -29,20 +29,20 @@ public function onError($code, $message, $file, $line)
         switch ($code) {
             case E_NOTICE:
             case E_USER_NOTICE:
-                $exc = new Errors\Notice($message, $file, $line, debug_backtrace());
+                $exc = new Errors\Notice($message, '', 0, debug_backtrace());
```

Comment:
Because this is what error handler accepts: http://php.net/manual/ru/function.set-error-handler.php

https://github.com/airbrake/phpbrake/blob/fix/error-handler-test/src/ErrorHandler.php#L72 is a completely different function that uses different PHP API.